src/creature.h

```diff
@@ -186,6 +205,15 @@ class Creature
                                                    double total_dispersion );
 
         /**
+         * Makes an aiming/attack roll for a single projectile attack shot.
+         *
+         * @param dispersion the base dispersion of the projectile attack
+         * @param range the range of the projectile attack
+         * @return the results of the aiming roll
```

Comment:
The extra definitions here don't add much. Doxygen comments aren't obligatory when they don't add any relevant information